res/css/structures/_UserMenu.scss

```diff
@@ -87,7 +88,8 @@ limitations under the License.
             flex-direction: column;
             width: calc(100% - 40px); // 40px = 32px theme button + 8px margin to theme button
 
-            * {
+            .mx_UserMenu_contextMenu_displayName,
+            .mx_UserMenu_contextMenu_userId {
                 // Automatically grow all subelements to fit the container
```

Comment:
Based on this comment it is expected to hide display name and user ID over the width